platform/linux-generic/pktio/stats/packet_io_stats.c

```diff
@@ -12,10 +12,28 @@
 
 #include <string.h>
 
+static int _odp_sock_stats_get(pktio_entry_t *e, odp_pktio_stats_t *stats,
```

Comment:
`_odp_sock_stats_get()` is internal helper, so `_odp_` prefix could be dropped.

Reply:
Changed to sock_stats_get() in v3.